controllers/resource/template.go

```diff
@@ -211,8 +211,6 @@ func (r *CloudStackTemplate) TemplateResources(ctx context.Context, eksaCluster
 
 	cpOpt := func(values map[string]interface{}) {
 		values["controlPlaneTemplateName"] = controlPlaneTemplateName
-		values["cloudstackControlPlaneSshAuthorizedKey"] = sshAuthorizedKey(cpCsmc.Spec.Users)
-		values["cloudstackEtcdSshAuthorizedKey"] = sshAuthorizedKey(etcdCsmc.Spec.Users)
```

Comment:
Moving these to the template builder where there is now one logical way assigning the keys to the MachineTemplate